source/03-components/external-link/external-link.es6.ts

```diff
@@ -0,0 +1,52 @@
+import Drupal from 'drupal';
+import once from 'once';
+
+Drupal.behaviors.externalLink = {
+  attach(context, settings) {
+    const { imagePath } = settings.gesso;
+    const exitDisclaimer = settings?.gesso?.externalLinkExitDisclaimer ?? 'Exit this website';
+    const allowedDomains = settings?.gesso?.externalLinkAllowedDomains ?? [];
+    const allowedLinks = settings?.gesso?.externalLinkAllowedLinks ?? [];
+    const externalLinks: NodeListOf<HTMLAnchorElement> = once(
+      'external-link',
+      "a:not([data-not-external-link], [href=''], [href^='#'], [href^='?'], [href^='/'], [href^='.'], [href^='javascript:'], [href^='mailto:'], [href^='tel:'])",
+      context
+    );
+
+    function linkIsExternal(link: HTMLAnchorElement) {
+      let external = true;
+
+      if (
+        link.host === window.location.host
+      ) {
+        external = false;
+      }
+
+      allowedDomains.forEach(domain => {
+        if (
+          link.host === domain ||
+          link.host.endsWith(`.${domain}`)
+        ) {
+          external = false;
+        }
+      });
+
+      if (allowedLinks.includes(link.href.toLowerCase())) {
+        external = false;
+      }
+
+      return external;
+    }
+
+    externalLinks.forEach(link => {
+      if (link.hasAttribute('href') && linkIsExternal(link)) {
+        const accessibleLabel = Drupal.t(exitDisclaimer);
```

Comment:
I believe the `Drupal.t()` needs to go around `exitDisclaimer` where it's defined, so you're translating a string, not a variable. I find the documentation confusing, but I think that's what https://www.drupal.org/docs/8/api/translation-api/overview#s-translation-in-javascript-files is saying

Reply:
Good catch! I pushed up this fix.